packages/theme-toolkit/src/component-stories-util.js

```diff
@@ -0,0 +1,16 @@
+export const STORY_GROUPS = {
```

Comment:
I imagine this is usefull to use in both utrecht story definitions and den haag story definitions, so both a default utrecht button and a default den haag button could be grouped in 1... 🤷‍♀️